src/builder.ts

```diff
@@ -157,6 +157,17 @@ export interface Options {
    */
   scalars?: Partial<Record<Typegen.GetGen<'scalars', string>, GraphQLScalarType>>
   computedInputs?: GlobalComputedInputs
+  /**
+   * Enable atomic operations
+   *
+   * @remarks
+   *
+   * GraphQL doesn't support union types. The plugin has to apply a flattening heuristic and pick by default the broadest member of the union.
+   * On update and upsert, the broadest member is the atomic operation, which is not ideal in many cases.
```

Comment:
I don't really understand this. What does "broadest member" mean? Is this point about when atomic operations are `false` or when they are `true`?